src/src/Controller/ContactsController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller;
+use App\Entity\MessageField;
+use App\Form\MessageFieldType;
+use Symfony\Component\HttpFoundation\Request;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class ContactsController extends AbstractController
+{
+    #[Route('/contacts', name: 'app_contacts')]
```

Comment:
On some pages, you have the path (e.g. `/contacts`) in English, on some other in Czech. Ideally, you should stick with one.